edk2-rockchip/Platform/Radxa/CM3/CM3.dsc

```diff
@@ -264,6 +264,16 @@
 
   gArmTokenSpaceGuid.PcdArmGicV3WithV2Legacy|FALSE
 
+  #
```

Comment:
Please move these to the PcdsFixedAtBuild.common section.

Reply:
Agreed, this should be right before the "PCI support" section, if you diff it with Quartz64.dsc